_posts/2019-10-26-pyos-min-mistakes-theme.md

```diff
@@ -0,0 +1,298 @@
+---
```

Comment:
This is a blog post, but it can also serve as a "kitchen sink" documentation file that demonstrates how to add specific elements to a page that we use frequently. It can also serve as a testing page for our website.